src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -40,6 +43,9 @@ public static IApplicationBuilder UseApplicationInsightsExceptionTelemetry(this
 
         public static void AddApplicationInsightsTelemetry(this IServiceCollection services, IConfiguration config)
         {
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddSingleton<IActionContextAccessor, ActionContextAccessor>();
```

Comment:
Ok apparently ASP.NET doesn't add these, so ignore my comment.

---
In reply to: [59233587](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/142#discussion_r59233587) [](ancestors = 59233587)